modular_skyrat/modules/xenoarch/code/modules/research/xenoarch/xenoarch_item.dm

```diff
@@ -15,8 +15,8 @@
 		if(istype(src, /obj/item/xenoarch/useless_relic/magnified))
 			balloon_alert(user, "already magnified!")
 			return
-		if(!is_curator_job(user.mind?.assigned_role))
-			balloon_alert(user, "must be a curator!")
+		if(!is_curator_job(user.mind?.assigned_role) && !HAS_TRAIT(user, TRAIT_XENOARCH_QUALIFIED))
```

Comment:
If you're replacing it with a skillchip, I feel like they should have the skillchip when they join in the round (so it would have to be apart of their outfit, you can check the cooks for an example), so we don't need to check that they are a curator everywhere for this feature.